google/cloud/aiplatform/explain/metadata/metadata_builder.py

```diff
@@ -28,3 +28,7 @@ class MetadataBuilder(_ABC):
     @abc.abstractmethod
     def get_metadata(self):
         """Returns the current metadata as a dictionary."""
+
+    @abc.abstractmethod
+    def get_metadata_object(self):
```

Comment:
It seems like this can be informative if further qualified as ```get_metadata_protobuf``` or ```get_metadata_pb```.

Reply:
Thanks, changed to `get_metadata_protobuf`